massgov/openmass
# DP-22947: Changes to new organization page integration branch

**Description:**
Changes to the Organization Section paragraph and Organization content type form display.


**Jira:** (Skip unless you are MA staff)
DP-22947


**To Test:**
- Login and go to /node/add/org_page.
- Verify the "More" tab is now "Navigation menu".
- Click on the "Overview" tab.
- Verify the "Organization Parent" and "Billing Organization" fields are now at the bottom of this tab.
- Click on the "Content" tab.
- Verify the Organization Section paragraph no longer has an "Additional Resources" field below "Section Content".


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
